server/src/test/java/io/crate/integrationtests/PgCatalogITest.java

```diff
@@ -85,6 +88,25 @@ public void testPgNamespaceTable() {
             "NULL| sys| 0| -458336339");
     }
 
+    @Test
+    public void testPgNamespaceTable_shows_namespace_if_user_can_see_a_table() {
+        execute("create table vip.tbl (x int)");
+        execute("create user hoschi");
+        execute("grant dql on table doc.t1 to hoschi");
+
+        UserLookup userLookup = cluster().getInstance(UserLookup.class);
+        Sessions sessions = cluster().getInstance(Sessions.class);
+        try (var session = sessions.newSession("doc", userLookup.findUser("hoschi"))) {
+            execute("select nspname from pg_catalog.pg_namespace order by nspname", session);
+            // vip schema doesn't show up
```

Comment:
I'm confused. It states in the issue description that schema actually should be shown?

> The schema should be visible. 
> In this case Sisense expects to see the schema in pg_namespace


I tried reproduction steps locally and diff with the PR is only showing doc 
(probably because we created session with such default schema)

that's output on nightly for me when running crash from non-crate user:
```
select nspname from pg_catalog.pg_namespace;
+--------------------+
| nspname            |
+--------------------+
| information_schema |
| pg_catalog         |
+--------------------+
```

Reply:
the test ensures the doc schema is shown due to the privilege on the table,  but the vip schema isn't because of no privileges

Reply:
ah ok, here it's 2 different schemas, I missed that.